Sources/ApplicationSpecs/ViewSpec.swift

```diff
@@ -9,10 +9,67 @@ class ViewSpec: Swiftest.Spec {
 			io = MockIO()
        		 	view = View(io: io)
         	}
+
+	        describe("#promptNumberOfGuesses") {
+        	        it("should print a prompt requesting number of guesses") {
+	                	let outputToPrint = "How many guesses do you want to have for this game?"
+        	            	view.promptNumberOfGuesses()
+
+                	    	expect(io.displayCalled).to.equal(true)
+                    		expect(io.getPrintedOutputStream).to.equal(outputToPrint)
+                	}
+            	}
         
-       		describe("#receiveGuess") {
+       		describe("#receiveNumberOfGuesses") {
+                	var errorValue: String!
+               		before() {
+                    	errorValue = "9999"
```

Comment:
No. It has to be a string that can be converted to an integer, and something that is beyond a reasonable guess I guess.